azure-kusto-data/azure/kusto/data/_token_providers.py

```diff
@@ -201,6 +204,12 @@ def __enter__(self):
     def __exit__(self, exc_type, exc_val, exc_tb):
         self.close()
 
+    async def __aenter__(self):

```

Comment:
This is only for tests?

Reply:
I guess - but it's weird to implement aexit without aenter, they usually go together